egs/lre07/README.txt

```diff
@@ -21,7 +23,7 @@
    CALLFRIEND Taiwan Chinese Mandarin:   LDC96S56
    CALLFRIEND Caribbean Spanish:         LDC96S57
    CALLFRIEND Non-Caribbean Spanish:     LDC96S58
-   LRE 1996:                             LDC?
+   LRE 1996:                             lid96e1
```

Comment:
LRE 1996 is actually included with LDC2006S31, so you can replace lid96e1 with LDC2006S31.